packages/compiler-cli/ngcc/test/rendering/renderer_spec.ts

```diff
@@ -641,7 +641,8 @@ UndecoratedBase.ɵdir = ɵngcc0.ɵɵdefineDirective({ type: UndecoratedBase, vie
              expect(sourceFile.contents)
                  .toEqual(RENDERED_CONTENTS + '\n' + generateMapFileComment('file.js.map'));
              expect(mapFile.path).toEqual(_('/node_modules/test-package/src/file.js.map'));
-             expect(JSON.parse(mapFile.contents)).toEqual(MERGED_OUTPUT_PROGRAM_MAP.toObject());
+             expect(JSON.parse(mapFile.contents) as SourceMapConverter)
+                 .toEqual(MERGED_OUTPUT_PROGRAM_MAP.toObject());
```

Comment:
This one really should be `as any`, since that is the return type of `SourceMapConverter.toObject()`.

```suggestion
             expect(JSON.parse(mapFile.contents) as any)
                 .toEqual(MERGED_OUTPUT_PROGRAM_MAP.toObject());
```